Program.cs

```diff
@@ -13,11 +13,19 @@
 builder.Services.AddScoped<MemberAuthentication>();
 builder.Services.AddScoped<MemberAuthorization>();
 builder.Services.AddScoped<MemberRegistration>();
-builder.Services.AddAuthentication().AddJwtBearer(options => options.TokenValidationParameters = new TokenValidationParameters
-{
-    ValidIssuer = builder.Configuration["Jwt:Issuer"],
-    ValidAudience = builder.Configuration["Jwt:Audience"],
-    IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(builder.Configuration["Jwt:Key"]))
+
+builder.Services.AddAuthentication().AddJwtBearer(options => {
+    options.SaveToken = true;
```

Comment:
SaveToken seems redundant as we do not really on any of its functionalities

Reply:
agreed, removed